leftwm/src/config/checks.rs

```diff
@@ -100,8 +100,8 @@ impl Config {
                 match error.0 {
                     Some(binding) => {
                         println!(
-                            "\x1b[1;91mERROR: {} for keybind {:?}\x1b[0m",
-                            error.1, binding
+                            "\x1b[1;91mERROR: {} for keybind {binding:?}\x1b[0m",
```

Comment:
Interesting, why the `error.1` is not pulled in. Is this because of the `.1`?

Reply:
Just had a look at this, obviously the compiler does not process the tuple selection, when inside the curlies...